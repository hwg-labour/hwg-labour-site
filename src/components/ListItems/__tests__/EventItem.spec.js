import { EventItem } from "../index";
import React from "react";
import { shallow } from "enzyme";

const EVENT = {
  node: {
    image: {
      file: {
        url: "image.png"
      }
    },
    title: "Event Title",
    date: Date(),
    membersOnly: false,
    description: "dsjadhiuashdiashidhasd"
  }
};

describe("EventItem", () => {
  it("renders component", () => {
    const wrapper = shallow(<EventItem event={EVENT} />);

    expect(wrapper).toMatchSnapshot();
  });
});
