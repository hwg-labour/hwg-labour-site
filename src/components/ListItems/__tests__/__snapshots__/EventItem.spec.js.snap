// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EventItem renders component 1`] = `
<GridRow
  key="undefined-newsitem"
>
  <GridColumn>
    <EventItem__EventThumbnail
      as={[Function]}
      src="https://res.cloudinary.com/codogo/image/fetch/w_800,c_fill,g_face,f_auto/https:
					image.png"
      to="/events/event-title"
    />
  </GridColumn>
  <GridColumn>
    <GatsbyLink
      to="/events/event-title"
    >
      <Header
        as="h3"
      >
        Event Title
      </Header>
    </GatsbyLink>
    <p
      style={
        Object {
          "color": "#aaaaaa",
        }
      }
    >
      March 9th 2020
    </p>
    <p>
      dsjadhiuashdiashidhasd
    </p>
    <Button
      as={[Function]}
      size="small"
      to="/events/event-title"
    >
      Read more 
      <Icon
        as="i"
        name="right arrow"
      />
    </Button>
  </GridColumn>
  <EventItem__NewsDivider
    section={true}
  />
</GridRow>
`;
